test/functional/Java8andUp/src/org/openj9/test/contendedfields/ContendedFieldsTests.java

```diff
@@ -63,7 +65,8 @@ static void getVMConfig() { /* ideally we should have an mxbean to get this info
 			CACHE_LINE_SIZE = 64;			
 		}
 		jep142Restricted = true;
-		for (String vmarg: ManagementFactory.getRuntimeMXBean().getInputArguments()) {
+		RuntimeMXBean bean = (RuntimeMXBean)ManagementFactory.getRuntimeMXBean();
```

Comment:
`-XX:-RestrictContended` is a user argument, so it should show up in the regular `getInputArguments()`.